modules/nf-core/porechop/porechop/main.nf

```diff
@@ -22,12 +22,16 @@ process PORECHOP_PORECHOP {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
+    ## To ensure ID matches rest of pipeline based on meta.id rather than input file name
+    ln -s $reads ${prefix}.fastq.gz
+
     porechop \\
-        -i $reads \\
+        -i ${prefix}.fastq.gz \\
         -t $task.cpus \\
         $args \\
-        -o ${prefix}.fastq.gz \\
+        -o ${prefix}_porechopped.fastq.gz \\
```

Comment:
Shall we update the nf-core module instead? https://github.com/nf-core/modules/blob/master/modules/nf-core/porechop/porechop/main.nf ?

Reply:
Yes we should... at some point ;)